doc/source/release_notes_and_installation.rst

```diff
@@ -11,4 +11,4 @@ Release notes
 Installation
 ============
 
-.. include:: install.rst
+.. include:: install.md
```

Comment:
there seems to be a problem here, see https://114-2014929-gh.circle-artifacts.com/0/doc/build/html/release_notes_and_installation.html#installation

Reply:
Another good catch - this seems to stem from the same cause as the problem in the above comment (`include`-ing an .rst formatted file from outside of the docs build inside a markdown document). Reverted back to rST in ef33c3a